package.json

```diff
@@ -143,5 +143,8 @@
     "hooks": {
       "pre-push": "node ./bin/devkit-admin hooks/pre-push"
     }
+  },
+  "resolutions": {
+    "browserslist": "4.5.5"
```

Comment:
Now that this has _fixed_ yarn's lock file with the newer version, this is not actually needed.

Reply:
Removed